apps/mull-ui/src/app/pages/direct-message/direct-message.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { ChatHeader, CustomTextInput } from '../../components';
+import ContactRow from '../../components/contact-row/contact-row';
+import './direct-message.scss';
+
+/* eslint-disable-next-line */
+export interface DirectMessageProps {}
+
+// TODO: Query that returns an array of user profiles related to user
+const userData = [
+  {
+    id: 1,
+    name: 'Bob Marley',
+    description: 'Singer',
+    picture:
+      'https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg',
+  },
+  {
+    id: 2,
+    name: 'Shawn Mendes',
+    description: 'Singer',
+    picture:
+      'https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg',
+  },
+  {
+    id: 3,
+    name: 'Justin Bieber',
+    description: 'Singer',
+    picture:
+      'https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg',
+  },
+];
+
+const DirectMessagePage = () => {
+  const [searchValue, setSearchValue] = useState('');
+
+  return (
+    <div className="direct-messages-container">
+      <ChatHeader eventTitle="Direct Messages" />
```

Comment:
To stay consistent with the Announcements/Group Chat, I believe that the hamburger menu and the Direct Messages title should be with the top nav-bar like in the picture contrary to what's on Figma.

![image](https://user-images.githubusercontent.com/38992971/109441506-6735c600-7a03-11eb-8e5b-001fda2a00d0.png)

```suggestion
      <ChatHeader eventTitle="Direct Messages" className="top-nav-bar-shadow" />
```